tests/tests_api/test_personal_info.py

```diff
@@ -29,7 +29,7 @@ def test_personal_info_create(api_client, fake_personal_info_data):
 
 
 def test_personal_info_update(api_client, fake_personal_info_data):
-    personal_info = baker.make(PersonalInfo)
+    personal_info = baker.make(PersonalInfo, first_name=seq("Ivan"))
```

Comment:
I would add a CSI-style comment saying that we're doing the sequence for `first_name` only because that will be enough for a `UniqueConstraint` to never be violated even if `last_name` and `email` are the same.

Also, this "Ivan" is repeated many times and is maybe worth putting into a constant (although I know it's not that common in tests)